tests/src/test/java/alluxio/client/fs/ConcurrentFileSystemMasterCreateTest.java

```diff
@@ -321,7 +323,12 @@ private void runLoadMetadata(WriteType writeType, boolean useSinglePath, boolean
               paths, limitMs);
     }
     if (!errors.isEmpty()) {
-      Assert.fail("Encountered " + errors.size() + " errors, the first one is " + errors.get(0));
+      StringWriter sw = new StringWriter();
+      PrintWriter pw = new PrintWriter(sw);
+      errors.get(0).printStackTrace(pw);
+
+      Assert.fail("Encountered " + errors.size() + " errors, the first one is " + errors.get(0)
```

Comment:
FYI guava has a utility for this, I think it's called `Throwables.getStackTrace(Throwable t)`

Reply:
oh cool. done